be_wmg_aws/lift_studies/utils/data_utils.py

```diff
@@ -12,7 +12,8 @@ class LiftS3Handler:
     """
 
     def __init__(self):
-        self.client = boto3.client("s3", region_name="sa-east-1")
+        self.region = os.environ['AWS_REGION']
```

Comment:
Please add linter exception to satisfy Meta's internal linter:

self.region = os.environ["AWS_REGION"]  # noqa: F821